src/test/java/PlayASongTests.java

```diff
@@ -0,0 +1,61 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PlayASongTests extends BaseTest {
+
+    String email = "holostenco.yuliya@gmail.com";
+    String searchPhrase = "Waiting On A Train";
+
+    @Test
+    public void playSong() throws InterruptedException {
+
+        login();
+        search();
+        playSelectedSong();
+        validatePlayOfTheSong();
+
+
+    }
+
+    private void validatePlayOfTheSong() {
+        WebElement imgOfAPlayingSong = driver.findElement(By.xpath("//img[@alt='Sound bars']"));
+        Assert.assertTrue(imgOfAPlayingSong.isDisplayed());
```

Comment:
For the stable test, you can add `Thread.sleep(2000);`

![image](https://user-images.githubusercontent.com/104761482/198562844-8bf6ff1c-1e46-4968-b6fb-388be9f673ab.png)